src/main/java/com/yelp/nrtsearch/server/luceneserver/index/ImmutableIndexState.java

```diff
@@ -162,6 +162,7 @@ public class ImmutableIndexState extends IndexState {
           .setDefaultSearchTimeoutSec(DoubleValue.newBuilder().setValue(0).build())
           .setDefaultSearchTimeoutCheckEvery(Int32Value.newBuilder().setValue(0).build())
           .setDefaultTerminateAfter(Int32Value.newBuilder().setValue(0).build())
+          .setMaxMergePreCopyDurationSec(DoubleValue.newBuilder().setValue(0))
```

Comment:
This is the default value used when the field is not specified in the committed state. There are a few other things to do here:

- Create a class field below to hold the instance value
- Add validation to `validateLiveSettings` method
- Assign instance value in the constructor
- Add abstract getter method to `IndexState`
- Implement method in this class to get instance value (maybe just return a sane default for `LegacyIndexState`)